test/pkgs/PythonEnginePackage/extra/DSPythonNet3.deps.json

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
I don't think you should commit this, where did it even come from?

Reply:
Came from the test package I added, will remove this file.